pipelines/edx-sysadmin-publish.yml

```diff
@@ -1,55 +1,56 @@
 ---
 resource_types:
-- name: pypi
-  type: docker-image
-  source:
-    repository: cfplatformeng/concourse-pypi-resource
-    tag: latest
+  - name: pypi
+    type: docker-image
+    source:
+      repository: cfplatformeng/concourse-pypi-resource
+      tag: latest
 
 resources:
-    - name: edx-sysadmin-repo
-      type: git
-      source:
-          uri: https://github.com/mitodl/edx-sysadmin.git
-          branch: release
-
-    - name: edx-sysadmin-pypi
-      type: pypi
-      source:
-        name: edx-sysadmin
-        packaging: any
-        repository:
-          username: ((pypi_creds.username))
-          password: ((pypi_creds.password))
+  - name: edx-sysadmin-repo
+    type: git
+    source:
+        uri: https://github.com/mitodl/edx-sysadmin.git
+        branch: release
+  - name: edx-sysadmin-pypi
+    type: pypi
+    source:
+      name: edx-sysadmin
+      packaging: any
+      repository:
+        username: ((pypi_creds.username))
+        password: ((pypi_creds.password))
 
 jobs:
-- name: publish-edx-sysadmin-to-pypi
-  public: true
-  plan:
-  - get: edx-sysadmin
-    trigger: true
+  - name: publish-edx-sysadmin-to-pypi
+    public: true
+    plan:
+    - get: edx-sysadmin-repo
+      trigger: true
 
-  - task: build-and-upload
+    - task: build-and-upload
       config:
         platform: linux
         image_resource:
           type: docker-image
           source:
             repository: python
         inputs:
-          - name: edx-sysadmin
+          - name: edx-sysadmin-repo
         outputs:
           - name: edx-sysadmin/dist
         run:
-          path: python
+          path: sh
           args:
-            - setup.py
-            - bdist_wheel
-            - sdist
+            - -exc
+            - |
+              cd edx-sysadmin
```

Comment:
There's a parameter for the run command that sets the directory that it executes in so that you don't have to run `cd` as part of the task. You can just add `dir: edx-sysadmin-repo` to the `run` block.